src/peoplemeasurement/forms.py

```diff
@@ -1,107 +1,97 @@
 from django import forms
-from django.contrib.gis.geos import LineString, Polygon
+from .serializers import AreaSerializer, LineSerializer
 
-from peoplemeasurement.models import Area, Sensors
+from peoplemeasurement.models import Area, Line
 
 
-class AreaForm(forms.ModelForm):
-    json_input = forms.JSONField(required=False)
+class BaseForm(forms.ModelForm):
+    def clean_coordinates(self) -> list:
+        coordinates = self.cleaned_data.get('coordinates')
+        if not coordinates:
+            return []
+        try:
+            self.validate_coordinates(coordinates)
+        except forms.ValidationError:
+            raise
+        except Exception:
+            raise forms.ValidationError("The coordinates cannot be interpreted. Format the input like the example")
```

Comment:
Je zou hier `raise from` kunnen gebruiken zodat de oorspronkelijke exception eventueel nog beshikbaar is...

```
except Exception as e:
    msg = "The coordinates cannot be interpreted. Format the input like the example"
    raise forms.ValidationError(msg) from e
```